src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProvider.cs

```diff
@@ -128,11 +131,12 @@ public override async Task<string> OnGetUnevaluatedPropertyValueAsync(string pro
 
         private static async Task<ComplexTargetFramework> GetStoredComplexTargetFrameworkAsync(ConfigurationGeneral configuration)
         {
-            ComplexTargetFramework storedValues = new ComplexTargetFramework
+            var storedValues = new ComplexTargetFramework
```

Comment:
can you keep as ComplexTargetFramework  instead of var?

Reply:
This was the suggestion given from the warning [IDE0007](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008), since it already declares it's going to be a `ComplexTargetFramework` variable.

Reply:
Yeah. The rules are set up where if RHS (right-hand side) of the expression is a known type, it'll suggest `var`. Else, suggest a type name. `new`ing a type is the main scenario where `var` is used.